packages/flutter/lib/src/widgets/image.dart

```diff
@@ -240,6 +234,8 @@ class _ImageState extends State<Image> {
 
   @override
   Widget build(BuildContext context) {
+    if (_imageStream == null)
+      _resolveImage();
```

Comment:
Yes it would. In this case it's only happening once, since _resolveImage inits _imageStream.